OutOfSchool/OutOfSchool.WebApi.Tests/Services/Database/WorkshopServiceTests.cs

```diff
@@ -864,6 +838,27 @@ private void SetupGetByFilter(IEnumerable<Workshop> workshops, IEnumerable<Avera
                 .Select(w => new WorkshopCard() { ProviderId = w.ProviderId, WorkshopId = w.Id, }).ToList());
     }
 
+    private List<Application> SetupApplications(Workshop workshop, int approvedApplications)
+    {
+        var allApplications = approvedApplications + 3;
+        var applications = ApplicationGenerator.Generate(allApplications)
+            .WithWorkshop(workshop)
+            .WithParent(ParentGenerator.Generate())
+            .WithChild(ChildGenerator.Generate());
+        for (int i = 0; i < allApplications; i++)
+        {
+            if (i < approvedApplications)
+            {
+                applications[i].Status = ApplicationStatus.Approved;
+            }
+            else
+            {
+                applications[i].Status = ApplicationStatus.Rejected;
+            }
```

Comment:
(Code style, suggestion) Not insisting, but consider avoiding heavy `if / else` pattern, when you are doing same things in both `if` and `else` branches. Here only `.Status` is set - so it's possible to have one assignment like

`applications[i].Status = i < approvedApplications ? ApplicationStatus.Approved : ApplicationStatus.Rejected;`

Reply:
Fixed. Thank you!